zulipterminal/ui_tools/views.py

```diff
@@ -1604,26 +1604,12 @@ def __init__(
         # computing their slice indexes
         button_widgets = []  # type: List[Any]
 
-        if topic_links:
-            topic_link_widgets, topic_link_width = self.create_link_buttons(
-                controller, topic_links
-            )
-
-            # slice_index = Number of labels before topic links + 1 newline
-            #               + 1 'Topic Links' category label.
-            slice_index = len(msg_info[0][1]) + 2
-            slice_index += sum([len(w) + 2 for w in button_widgets])
-            button_widgets.append(topic_links)
-
-            widgets = widgets[:slice_index] + topic_link_widgets + widgets[slice_index:]
-            popup_width = max(popup_width, topic_link_width)
-
         if message_links:
             message_link_widgets, message_link_width = self.create_link_buttons(
                 controller, message_links
             )
 
-            # slice_index = Number of labels before message links + 1 newline
+            # slice_index = Number of labels before message links + 1 newlineSS
```

Comment:
It's useful to review the commit using your own local tools (I use `tig` on the command line) or here on GitHub to find things like this which sneak in by accident :)

Reply:
Thank you for the excellent advice! I`ll fix the problem and look into testing.